scrapy_zyte_api/responses.py

```diff
@@ -75,11 +96,12 @@ def from_api_response(cls, api_response: Dict, *, request: Request = None):
             raw_api_response=api_response,
         )
 
+    def replace(self, *args, **kwargs):
+        kwargs.setdefault("encoding", self.encoding)
+        return ZyteAPIMixin.replace(self, *args, **kwargs)
```

Comment:
It seems super() is going to work here, but I don't see an issue with having explicit ZyteAPIMixin as well.